src/components/CustomAuthInput.jsx

```
@@ -0,0 +1,40 @@
+  label,
+  name,
+  type = 'text',
+  placeholder,
+  value,
+  onChange,
+  required = false,
+  error,
+}) {
+  return (
+    <div className="flex flex-col">
+      <label className="text-sm font-semibold text-gray-700">{label}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)